test/org/sagebionetworks/bridge/play/controllers/ParticipantControllerTest.java

```diff
@@ -257,23 +272,28 @@ public void updateParticipantRequiresIdMatch() throws Exception {
     public void getSelfParticipant() throws Exception {
         StudyParticipant studyParticipant = new StudyParticipant.Builder().withFirstName("Test").build();
         
-        when(participantService.getParticipant(STUDY, NO_CALLER_ROLES, ID)).thenReturn(studyParticipant);
+        when(participantService.getParticipant(study, ID, false)).thenReturn(studyParticipant);
 
         Result result = controller.getSelfParticipant();
         
-        verify(participantService).getParticipant(STUDY, NO_CALLER_ROLES, ID);
+        verify(participantService).getParticipant(study, ID, false);
         
         StudyParticipant deserParticipant = BridgeObjectMapper.get()
                 .readValue(Helpers.contentAsString(result), StudyParticipant.class);
 
         assertEquals("Test", deserParticipant.getFirstName());
+        assertNull(deserParticipant.getHealthCode());
```

Comment:
Since we explicitly clear out both health code and encrypted health code, we should also explicitly check both health code and encrypted health code here.